### ISSUE TITLE
setData in BoltFormsEvent

### ISSUE THREAD

**Issue by Spidlace** [bobdenotter/forms#52]
Hello !

I want to add event listener on pre_submit (for add data before submit) in a form.
I try to listen BoltFormsEvents::PRE_SUBMIT or boltforms.pre_submit (same) but when I try setData from Bolt\BoltForms\Event, I got an error : 

> An exception has been thrown during the rendering of a template ("Bolt\BoltForms\Event\BoltFormsEvent::setData can only be called in BoltFormsEvents::PRE_SUBMIT")

I thinks the problem is vendor/bolt/forms/src/Event/BoltFormsEvent.php (l.65)

```php
if ($this->formsEventName === FormEvents::PRE_SUBMIT) {
  $this->event->setData($data);
} else {
  throw new \RuntimeException(self::class . '::' . __FUNCTION__ . ' can only be called in BoltFormsEvents::PRE_SUBMIT');
}
```

`$this->formsEventName` send BoltFormsEvents::PRE_SUBMIT but condition allow only FormEvents::PRE_SUBMIT

Can you check that or tell me if I'm wrong :-) 
(I can make a pull request if you want)

**Comment by I-Valchev** [comment]
@Spidlace you are indeed correct. Does it work if you expand the `if` statement to also accept `BoltFormsEvents::PRE_SUBMIT`?

If so, we can fix it and tag a release so you can use it freely :-) Otherwise, if you can create a PR that fixes it in another way, that's also fine!

**Comment by Spidlace** [comment]
Hi @I-Valchev 

Yes it works if I expand `if` statement
I created new PR, I thinks other people can be confronted to this problem :-)
Tell me if this PR works for you (It's ok for me)

**Comment by bobdenotter** [comment]
Fixed in #53
